nixos/doc/manual/default.nix

```diff
@@ -72,6 +73,63 @@ let
       </toc>
     '';
 
+  manualXsltprocOptions = toString [
```

Comment:
Reverted in c36d6e59647794aa1c4dbb2dcd7e5242d9c5b6b8 and brought back in #14700.